src/toil/cwl/cwltoil.py

```diff
@@ -3538,22 +3585,23 @@ def __init__(
         self.bypass_file_store = bypass_file_store
         self.import_data = import_data
 
-    def run(self, file_store: AbstractFileStore) -> Tuple[Process, CWLObjectType]:
-        """
-        Convert the filenames in the workflow inputs into the URIs
-        :return: Promise of transformed workflow inputs. A tuple of the job order and process
-        """
-        candidate_to_fileid: dict[str, FileID] = unwrap(self.import_data)
-
-        initialized_job_order = unwrap(self.initialized_job_order)
-        tool = unwrap(self.tool)
-
+    @staticmethod
+    def convert_files(
+        initialized_job_order: CWLObjectType,
+        tool: Process,
+        candidate_to_fileid: dict[str, FileID],
+        basedir: str,
+        skip_remote: bool,
+        bypass_file_store: bool,
+    ) -> tuple[Process, CWLObjectType]:
```

Comment:
I think this is missing a docstring.

It also might need a better name. It sounds like it is going to e.g. convert files from DOC to PDF or something, and really it is going to fill in Toil file IDs and replace URLs with other URLs.